### PR TITLE
macOS fixes, model curation, etc.

### PR DESCRIPTION
**Pull request by olupton**
- fix `--virtual` on macOS
- support fetching models from forked repositories on GitHub
- disable build parallelism internal to `nrnivmodl`
- do some more model curation

### CODE REVIEW

**Comment by olupton** [comment]
Launched https://github.com/neuronsimulator/nrn-modeldb-ci/actions/runs/4281622878 with all models.

File: modeldb/modeldb-run.yaml [hunk 1]
```diff
@@ -625,13 +631,16 @@
     run:
     - xopen("show_start.hoc")
     - verify_graph_()
+    script:
+    # avoid printing out times
+    - sed -i'.bak' -e 's/^startsw()$/not_used=startsw()/g' ga_setup.hoc
```
Comment (pramodk) [inline modeldb/modeldb-run.yaml:636]: a minor : `{startsw()}` works?

File: modeldb/modelrun.py [hunk 2]
```diff
@@ -134,7 +135,25 @@ def clean_model_dir(model):
 
 
 def compile_mods(model, mods):
-    run_commands(model, ["nrnivmodl"] + mods, work_dir=model.run_info["start_dir"])
+    # Unfortunately nrnivmodl doesn't have an option to steer how much build
```
Comment (pramodk) [inline modeldb/modelrun.py:138]: just a note: `nrnivmodl-core` has such option. May be we could add back port it in nrnivmodl.

File: modeldb/modeldb-run.yaml [hunk 3]
```diff
@@ -1086,6 +1102,14 @@
 223962:
     skip: true
     comment: takes too long, need to see how to reduce time
+229276:
+    model_dir: mechanisms
+    script:
+    - cp template.hoc template.hoc.iconv.bak
+    - cp createsimulation.hoc createsimulation.hoc.iconv.bak
+    - iconv -f LATIN1 -t UTF-8 template.hoc.iconv.bak > template.hoc
+    - iconv -f LATIN1 -t UTF-8 createsimulation.hoc.iconv.bak > createsimulation.hoc
```
Comment (pramodk) [inline modeldb/modeldb-run.yaml:1111]: just curious - do you think we should ask Rui to backport such changes into respective models?

**Review by pramodk — APPROVED** [review]
LGTM

**Comment by olupton** [comment]
Launched https://github.com/neuronsimulator/nrn-modeldb-ci/actions/runs/4282403327 with [b5ca035](https://github.com/neuronsimulator/nrn-modeldb-ci/pull/76/commits/b5ca03518b1b283a490dbb003fa460e2131ed3b0) and all models.

**Review by alexsavulescu — APPROVED** [review]
LGTM, already discussed these offline

**Comment by olupton** [comment]
> Launched https://github.com/neuronsimulator/nrn-modeldb-ci/actions/runs/4282403327 with [b5ca035](https://github.com/neuronsimulator/nrn-modeldb-ci/pull/76/commits/b5ca03518b1b283a490dbb003fa460e2131ed3b0) and all models.

This doesn't yet show a trivial diff, but I will merge this PR as it is already approved/reviewed and is a step in the right direction.

**Pull request merged**